nsqd/channel_test.go

```diff
@@ -152,7 +152,8 @@ func TestChannelEmptyConsumer(t *testing.T) {
 	channel := topic.GetChannel("channel")
 	client := newClientV2(0, conn, &context{nsqd})
 	client.SetReadyCount(25)
-	channel.AddClient(client.ID, client)
+	err := channel.AddClient(client.ID, client)
+	test.Equal(t, err, nil)
```

Comment:
this reminded me, it should be pretty easy to add a new test that validates behavior when the max is exceeded

Reply:
Will do.